packages/android_studio.rb

```diff
@@ -0,0 +1,48 @@
+require 'package'
+
+class Android_studio < Package
+  description 'Android Studio is the official IDE for Android development.'
+  homepage 'https://developer.android.com/studio'
+  version '3.3.1.0'
+  source_url 'https://dl.google.com/dl/android/studio/ide-zips/3.3.1.0/android-studio-ide-182.5264788-linux.zip'
```

Comment:
... This is not a good idea. You should at least fail out before downloading on other architectures.

Reply:
http://tools.android.com/build/studio